integration/README.md

```diff
@@ -0,0 +1,27 @@
+# Integration tests
+
+Here we have a basic integration test for the happy path.
+
+## Requirements
+
+You need these tools installed locally:
+
+- [kind](https://kind.sigs.k8s.io/)
+- [apptestctl](https://github.com/giantswarm/apptestctl/)
+
+## Running the tests
+
+Create the `kind` cluster and use `apptestctl` to bootstrap GiantSwarm's App platform.
+
+```bash
+(kind delete cluster || true) && kind create cluster && apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
```

Comment:
It would be nice to specify a name to the kind cluster in order to avoid accidentally deleting other kind clusters running in users' machines